eZ/Publish/Core/Persistence/Cache/ContentTypeHandler.php

```diff
@@ -56,11 +56,11 @@ protected function init(): void
                 'type-' . $type->id,
             ];
         };
-        $this->getTypeKeys = static function (Type $type, int $status = Type::STATUS_DEFINED) {
+        $this->getTypeKeys = function (Type $type, int $status = Type::STATUS_DEFINED) {
```

Comment:
Why removing `static`?

Reply:
to use the escapeForCacheKey function we need to change from a static function to normal